.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Description
+
+_Thanks for your contribution! Describe your changes in a few sentences. Try to include discussion of tradeoffs or alternatives you considered when writing this code._
```

Comment:
Maybe use `<!--` ... `-->` instead, so it doesn't show up in the preview view by default?